SCP fight club/Assets/Scripts/Punch.cs

```diff
@@ -5,14 +5,23 @@
 public class Punch : MonoBehaviour
 {
```

Comment:
It is hard to understand what this script does without comments, please explain it in the script